src/main/java/com/samoilenko/onlinebookstore/model/Book.java

```diff
@@ -10,11 +10,15 @@
 import lombok.EqualsAndHashCode;
 import lombok.Getter;
 import lombok.Setter;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
 
 @Entity
 @Getter
 @Setter
 @EqualsAndHashCode
+@SQLDelete(sql = "UPDATE books SET is_deleted = true WHERE id=?")
+@Where(clause = "is_deleted=false")
```

Comment:
where is deprecated so let's use `@SqlRestriction` instead, if it doesn't work - try to change the type of `is_deleted` column in liquibase to `BIT`